CMSgov/mint-app
# Feature/easi 3695 model participation questions

# [EASI-3695](https://jiraent.cms.gov/browse/EASI-3695) - FEATURE
### [EASI-3743 PR ](https://github.com/CMSgov/mint-app/pull/931) - FE PR

### [EASI-3741](https://jiraent.cms.gov/browse/EASI-3741) - BE Task

## Changes and Description

- Add fields related to participationInModelPrecondition
- Update Postman Collection



## How to test this change
1. Navigate to page 1 of General Characteristics
2. Fill out all the variations of the participationInModelPrecondition fields
3. Verify Data persists as expected


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
